sdk/go/aws/provider.go

```diff
@@ -86,7 +86,7 @@ type providerArgs struct {
 	// Configuration for retrieving temporary credentials from the STS service.
 	AssumeRole *ProviderAssumeRole `pulumi:"assumeRole"`
 	// Configuration block with resource tag settings to apply across all resources handled by this provider. This is designed to replace redundant per-resource `tags` configurations. Provider tags can be overridden with new values, but not excluded from specific resources. To override provider tag values, use the `tags` argument within a resource to configure new tag values for matching keys.
-	DefaultTags *ProviderDefaultTags `pulumi:"defaultTags"`
+	DefaultTags map[string]string `pulumi:"defaultTags"`
```

Comment:
Breaking change?

Reply:
Before last week this wasn't implemented. However, when doing the implementation it was done as a string map having not looked carefully enough at this schema. So since that went out, setting the properties via an explicit provider would have failed. So it's not going to break anyone.

However, it is an open question on if it would be better to change the provider behaviour to match this schema which would technically be a break for anyone who upgraded last week and set default tags via config.

Reply:
Ah very nice. Well perhaps there's some way to do a Configure/CheckConfig/etc hook to normalize to the desired form so it's certainly not breaking anyone.

Reply:
I have reverted the schema change and added a backward compatible parsing fix in the provider instead.